fractal_tasks_core/tables/v1.py

```diff
@@ -264,3 +264,58 @@ def _write_table_v1(
     table_group.attrs.update(**table_attrs)
 
     return table_group
+
+
+def get_tables_list_v1(
+    zarr_url: str, table_type: str = None, strict: bool = False
+):
+    """
+    Find the list of tables in the Zarr file
+
+    Optionally match a table type and only return the names of those tables.
+
+    Args:
+        zarr_url: Path to the OME-Zarr image
+        table_type (str): The type of table to look for. Special handling for
+            "ROIs" => matches both "roi_table" & "masking_roi_table".
+        strict (bool): If True, only return tables that have a type attribute.
+            If False, also include tables without a type attribute.
+    Returns:
+        table_list: list of the names of available tables
+    """
+    with zarr.open(zarr_url, mode="r") as zarr_group:
+        zarr_subgroups = list(zarr_group.group_keys())
+    if "tables" not in zarr_subgroups:
+        return [""]
```

Comment:
I cannot understand why returning `[""]` rather than `[]`. Is there any specific reason, or is this a typo?
(this line is not tested, anyway)

cc @jluethi

Reply:
A, good point! I think that's an artefact from using this in the napari plugin and that worked better for not showing options in the dropdown. Returning [] would be the better answer indeed!

Reply:
Fixed with 444abc2f43094f1bc310509b02e5ba1325d756eb